source/contracts/reporting/Reporting.sol

```diff
@@ -2,7 +2,7 @@ pragma solidity 0.4.24;
 
 
 library Reporting {
-    uint256 private constant DESIGNATED_REPORTING_DURATION_SECONDS = 3 days;
+    uint256 private constant DESIGNATED_REPORTING_DURATION_SECONDS = 1 days;
```

Comment:
Users will now have 24h instead of 72h from market endTime to submit a designated report. The impact here is that a market is almost 30% more likely to hit an earlier feeWindow for its first dispute round if designated reporter fails to show.